Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec rails server -p $PORT
+release: bundle exec rake db:migrate db:seed
```

Comment:
Connect Semaphore and let it handle that. It doesn't handle review apps although you can do that in the `app.json` file they use.